test/unit/resources/aws_ec2_instances_test.rb

```diff
@@ -0,0 +1,191 @@
+require 'helper'
```

Comment:
Really need some integration tests - under `test/integration/aws/default/verify/controls/`.  You may also need to add more EC2 instances to get enough variety to test the filters.  I'm happy to help.

Reply:
I'll have to check how you're doing those... might need to ping you for help.